src/services/handler/acknowledgement-handler.js

```diff
@@ -7,6 +7,8 @@ import { setCurrentSpanAttributes } from '../../config/tracing';
 import { logError, logInfo } from '../../middleware/logging';
 import { messageIdMatchOutboundCore } from '../database/dynamodb/ehr-core-repository';
 import { getNhsNumberByOutboundConversationId } from '../database/dynamodb/outbound-conversation-repository';
+import {updateConversationStatus, updateCoreStatus} from "../transfer/transfer-out-util";
+import {ConversationStatus, CoreStatus} from "../../constants/enums";
```

Comment:
Could we please add spaces there for consistency? 😄

```suggestion
import { updateConversationStatus, updateCoreStatus } from "../transfer/transfer-out-util";
import { ConversationStatus, CoreStatus } from "../../constants/enums";
```